jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -72,7 +74,12 @@ private void add(TarArchiveEntry tarArchiveEntry) {
       // directories.
       Path namePath = Paths.get(tarArchiveEntry.getName());
       if (namePath.getParent() != namePath.getRoot()) {
-        TarArchiveEntry dir = new TarArchiveEntry(DIRECTORY_FILE, namePath.getParent().toString());
+        TarArchiveEntry dir;
+        try {
+          dir = new TarArchiveEntry(DIRECTORY_FILE, namePath.getParent().toString());
+        } catch (IOException e) {
+          throw new UncheckedIOException(e);
```

Comment:
Just make this method `throws IOException`. I've verified that this is the simplest (and also right) change.

Reply:
done